pkg/database/mariadb.go

```diff
@@ -730,7 +730,12 @@ func getMysqlDumpBinlog(s string) (mp mysql.Position, err error) {
 		v := kv[2]
 		res[k] = v
 	}
-	pos, err := strconv.ParseInt(res["MASTER_LOG_POS"], 10, 32)
+	logPos, ok := res["MASTER_LOG_POS"]
+	if !ok {
+		return
```

Comment:
This will propagate the initial `mysql.Position` all the way back up to https://github.com/sapcc/maria-back-me-up/blob/38ef7073d89c280627028cec0c9a79fc98a2c23e/pkg/backup/manager.go#L202

Is the initial position valid to start syncing, or should this error out here?

Reply:
without it the tests are failing.
the scanner calls the `getMysqlDumpBinlog` twice. Second time the map is empty. Maybe something changed.
https://github.com/sapcc/maria-back-me-up/blob/11f57cd6b62e0342127b5a7a7b04b0586a24d92a/pkg/database/mariadb.go#L305